packages/techdocs-common/src/stages/publish/awsS3.ts

```diff
@@ -167,12 +167,12 @@ export class AwsS3Publish implements PublisherBase {
         const entityRootDir = `${entity.metadata.namespace}/${entity.kind}/${entity.metadata.name}`;
         const destination = `${entityRootDir}/${relativeFilePathPosix}`; // S3 Bucket file relative path
 
-        const fileContent = await fs.readFile(filePath, 'utf8');
+        const fileStream = fs.createReadStream(filePath);
 
```

Comment:
Thanks for this!

While we're doing this change, could you move the createReadStream into the limiter below? In the interest of not running out of inodes or something similarly esoteric, we probably don't want to create an unbounded number of simultaneous streams that we don't even intend to read from right away.

Reply:
Damn, that's a good point. Let me change.

Reply:
@freben Updated, please have a look.